scenarios/10_Siege_of_Haeltin.cfg

```diff
@@ -258,9 +277,72 @@
             [/have_unit]
             [then]
                 {MSG_Reumario _"We really should sacrifice an ox to God. There are so many of them; without God's help we won't be able to defeat them."}
+                {MSGW_Yahyazad _"We don't even have any oxen, though."}
+                {MSG_Reumario _"Well, something similar, then."}
+                [unit]
+                    type=Woodland Boar
+                    side=5
+                    id=sacrificial_boar
+                    x,y=4,4
+                    animate=yes
+                    [ai]
+                        [micro_ai]
+                            ai_type=forest_animals
+                            tusker_type=Woodland Boar
+                            [filter_location]
+                                terrain=*^F*,Gg,Hh,Re,Ss
+                            [/filter_location]
+                        [/micro_ai]
+                    [/ai]
+                [/unit]
+                {MSG_Reumario _"...like that boar! Let me go capture it, sacrifice it, and then surely God's favor will be upon us!"}
+                [event]
+                    name=last breath
+                    [filter]
+                        id=sacrificial_boar
+                    [/filter]
+                    [filter_second]
+                        id=Reumario
+                    [/filter_second]
+                    {MSG_Reumario _"I have caught it! Please, O God, accept this boar as a sacrifice, and look with blessing upon us, your children!"}
+                    {MSG_narrator _"Reumario feels reinvigorated by his performance of the sacrifice, and gains experience from it!"}
+                    [heal_unit]
+                        [filter]
+                            id=Reumario
+                        [/filter]
+                        amount=full
+                        animate=yes
+                        moves=full
+                        restore_attacks=yes
+                    [/heal_unit]
+                    {GIVE_UNIT_EXPERIENCE Reumario {ON_DIFFICULTY4 30 25 20 15}}
+                [/event]
+                [event]
+                    name=last breath
+                    [filter]
+                        id=sacrificial_boar
+                    [/filter]
+                    [filter_second]
+                        [not]
+                            id=Reumario
+                        [/not]
+                    [/filter_second]
+                    [if]
+                        [have_unit]
+                            id=Reumario
+                        [/have_unit]
+                        [then]
+                            {MSG_Reumario _"You fools! I was going to sacrifice that boar! But now you have killed it without performing the proper sacrificial ritual, and it will go to waste!"}
```

Comment:
```suggestion
                            {MSG_Reumario _"I was going to sacrifice that boar! But now you have killed it without performing the proper sacrificial ritual and it will go to waste!"}
```

Reply:
I really think the "You fools!" should stay here; he is *mad*